bazel/patches/wasmtime-0.20.0-fix-git-rev.diff

```diff
@@ -0,0 +1,21 @@
+diff --git a/crates/cache/src/lib.rs b/crates/cache/src/lib.rs
```

Comment:
You could move this to `bazel/external/wasmtime.patch` to colocate it with the `BUILD` file that uses it.